pkg/dashboard/ui/src/app/shared/services/nuclio-functions-data.service.js

```diff
@@ -23,9 +23,10 @@
          * Update existing function with new data
          * @param {Object} functionDetails
          * @param {string} projectName - the name of the project containing the function
+         * @param {boolean} importProcess - "true" if importing process
          * @returns {Promise}
          */
-        function createFunction(functionDetails, projectName) {
+        function createFunction(functionDetails, projectName, importProcess = false) {
```

Comment:
You're right that this is a good day to that in JavaScript.
But, in this code-base we do not use ES6 or later versions, because we started with ES5 and I didn't want to have to refactor the entire code-base to ES6.
Since “default function parameters” feature is not part of ES5, it was introduced only in ES6, please don't use it.
See my following comment on how to achieve this default value. At least one way to do it.